ClLike/cl_like/lpt.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, log10k_min=-4, log10k_max=2,
         else:
             self.wk_low = np.ones(nk_total)
 
-    def update_pk(self, cosmo):
+    def update_pk(self, cosmo, bcmpar={}):
```

Comment:
I would change `bcmpar` to `**kwargs` but we can do it in the future if we have to add more arguments

Reply:
Yeah, I agree, it would be more general. I'd change it to kwargs everywhere, but in that case let's just remember in bacco.py to create the appropriate bcmpar dictionary from the received kwargs

Reply:
We can keep `bcmpar` in bacco, apart from `kwargs`